library/src/methods/is/is.ts

```diff
@@ -0,0 +1,18 @@
+import { BaseSchema, Output } from '../../types';
+import { safeParse } from '../safeParse';
+
+/**
+ * Type guard for a given schema
+ *
+ * @param schema The scheme to be used
+ *
+ * @param input The input to be tested
+ *
+ * @returns A type predicate for the given schema
+ */
+export function is<TSchema extends BaseSchema>(
+  schema: TSchema,
+  input: unknown
+): input is Output<TSchema> {
+  return safeParse(schema, input).success;
```

Comment:
`safeParse` has more functionality and thus code than `is` technically required. Therefore we could change this line to the following code and remove the import above.

```ts
try {
  schema.parse(input);
  return true;
} catch (error) {
  return false;
}
```